content/pt-br/docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Configurando um Pod Para Usar um Volume Projetado Para Armazenamento
+content_type: task
+weight: 70
+reviewers:
+- jpeeler
+- pmorie
+---
+
+<!-- overview -->
+Esta página mostra como usar um Volume
+[projetado](/docs/concepts/storage/volumes/#projetado) para montar diversas 
```

Comment:
You can use point to the pt-br link: pt-br/docs/concepts/storage/volumes/#projetado